1.9/development/create-package.md

```diff
@@ -0,0 +1,153 @@
+---
+post_title: Creating a Universe Package
+nav_title: Creating a Universe Package
+menu_order: 10
+---
+
+This page covers general advice and information about creating a DC/OS package that can be published to the Mesosphere Universe. Consult the [Publish a Package][2] page of the Universe documentation for full details.
```

Comment:
This page is much more up-to-date...can we use it for the 1.8 docs also? Everything should be valid for 1.8.